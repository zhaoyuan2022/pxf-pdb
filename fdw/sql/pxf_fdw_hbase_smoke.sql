-- Create Hbase tables hbase_table and pxflookup
\!{{ HBASE_CMD }} shell {{ SCRIPT create_pxflookup.rb }} >/dev/null 2>&1
\!{{ HBASE_CMD }} shell {{ SCRIPT gen_small_data.rb }}

-- FDW test
CREATE SERVER h_base_smoke_test_server
	FOREIGN DATA WRAPPER hbase_pxf_fdw
	OPTIONS (config '{{ SERVER_CONFIG }}');
CREATE USER MAPPING FOR CURRENT_USER
	SERVER h_base_smoke_test_server;
CREATE FOREIGN TABLE h_base_smoke_test_foreign_table  (
		name text,
		num int,
		dub double precision,
		longnum bigint,
		bool boolean
	)
	SERVER h_base_smoke_test_server OPTIONS (resource 'hbase_small_data_table_{{ FULL_TESTNAME }}');

SELECT * FROM h_base_smoke_test_foreign_table ORDER BY name;
SELECT name, num FROM h_base_smoke_test_foreign_table WHERE num > 50 ORDER BY name;

-- clean up HBase
{{ CLEAN_UP }}\!{{ HBASE_CMD }} shell {{ SCRIPT drop_small_data.rb }} >/dev/null 2>&1
{{ CLEAN_UP }}\!rm -rf {{ SCRIPT drop_small_data.rb }} {{ SCRIPT gen_small_data.rb }} {{ SCRIPT create_pxflookup.rb }}
