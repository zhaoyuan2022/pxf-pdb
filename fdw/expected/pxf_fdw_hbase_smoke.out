-- Create Hbase tables hbase_table and pxflookup
\!{{ HBASE_CMD }} shell {{ SCRIPT create_pxflookup.rb }} >/dev/null 2>&1
\!{{ HBASE_CMD }} shell {{ SCRIPT gen_small_data.rb }}
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
0 row(s) in -.---- seconds
-- FDW test
CREATE SERVER h_base_smoke_test_server
	FOREIGN DATA WRAPPER hbase_pxf_fdw
	OPTIONS (config '{{ SERVER_CONFIG }}');
CREATE USER MAPPING FOR CURRENT_USER
	SERVER h_base_smoke_test_server;
CREATE FOREIGN TABLE h_base_smoke_test_foreign_table  (
		name text,
		num int,
		dub double precision,
		longnum bigint,
		bool boolean
	)
	SERVER h_base_smoke_test_server OPTIONS (resource 'hbase_small_data_table_{{ FULL_TESTNAME }}');
SELECT * FROM h_base_smoke_test_foreign_table ORDER BY name;
  name   | num |  dub   |    longnum     | bool 
---------+-----+--------+----------------+------
 row_1   |   1 |  1.001 |   100000000000 | f
 row_10  |  10 |  10.01 |  1000000000000 | t
 row_100 | 100 |  100.1 | 10000000000000 | t
 row_11  |  11 | 11.011 |  1100000000000 | f
 row_12  |  12 | 12.012 |  1200000000000 | t
 row_13  |  13 | 13.013 |  1300000000000 | f
 row_14  |  14 | 14.014 |  1400000000000 | t
 row_15  |  15 | 15.015 |  1500000000000 | f
 row_16  |  16 | 16.016 |  1600000000000 | t
 row_17  |  17 | 17.017 |  1700000000000 | f
 row_18  |  18 | 18.018 |  1800000000000 | t
 row_19  |  19 | 19.019 |  1900000000000 | f
 row_2   |   2 |  2.002 |   200000000000 | t
 row_20  |  20 |  20.02 |  2000000000000 | t
 row_21  |  21 | 21.021 |  2100000000000 | f
 row_22  |  22 | 22.022 |  2200000000000 | t
 row_23  |  23 | 23.023 |  2300000000000 | f
 row_24  |  24 | 24.024 |  2400000000000 | t
 row_25  |  25 | 25.025 |  2500000000000 | f
 row_26  |  26 | 26.026 |  2600000000000 | t
 row_27  |  27 | 27.027 |  2700000000000 | f
 row_28  |  28 | 28.028 |  2800000000000 | t
 row_29  |  29 | 29.029 |  2900000000000 | f
 row_3   |   3 |  3.003 |   300000000000 | f
 row_30  |  30 |  30.03 |  3000000000000 | t
 row_31  |  31 | 31.031 |  3100000000000 | f
 row_32  |  32 | 32.032 |  3200000000000 | t
 row_33  |  33 | 33.033 |  3300000000000 | f
 row_34  |  34 | 34.034 |  3400000000000 | t
 row_35  |  35 | 35.035 |  3500000000000 | f
 row_36  |  36 | 36.036 |  3600000000000 | t
 row_37  |  37 | 37.037 |  3700000000000 | f
 row_38  |  38 | 38.038 |  3800000000000 | t
 row_39  |  39 | 39.039 |  3900000000000 | f
 row_4   |   4 |  4.004 |   400000000000 | t
 row_40  |  40 |  40.04 |  4000000000000 | t
 row_41  |  41 | 41.041 |  4100000000000 | f
 row_42  |  42 | 42.042 |  4200000000000 | t
 row_43  |  43 | 43.043 |  4300000000000 | f
 row_44  |  44 | 44.044 |  4400000000000 | t
 row_45  |  45 | 45.045 |  4500000000000 | f
 row_46  |  46 | 46.046 |  4600000000000 | t
 row_47  |  47 | 47.047 |  4700000000000 | f
 row_48  |  48 | 48.048 |  4800000000000 | t
 row_49  |  49 | 49.049 |  4900000000000 | f
 row_5   |   5 |  5.005 |   500000000000 | f
 row_50  |  50 |  50.05 |  5000000000000 | t
 row_51  |  51 | 51.051 |  5100000000000 | f
 row_52  |  52 | 52.052 |  5200000000000 | t
 row_53  |  53 | 53.053 |  5300000000000 | f
 row_54  |  54 | 54.054 |  5400000000000 | t
 row_55  |  55 | 55.055 |  5500000000000 | f
 row_56  |  56 | 56.056 |  5600000000000 | t
 row_57  |  57 | 57.057 |  5700000000000 | f
 row_58  |  58 | 58.058 |  5800000000000 | t
 row_59  |  59 | 59.059 |  5900000000000 | f
 row_6   |   6 |  6.006 |   600000000000 | t
 row_60  |  60 |  60.06 |  6000000000000 | t
 row_61  |  61 | 61.061 |  6100000000000 | f
 row_62  |  62 | 62.062 |  6200000000000 | t
 row_63  |  63 | 63.063 |  6300000000000 | f
 row_64  |  64 | 64.064 |  6400000000000 | t
 row_65  |  65 | 65.065 |  6500000000000 | f
 row_66  |  66 | 66.066 |  6600000000000 | t
 row_67  |  67 | 67.067 |  6700000000000 | f
 row_68  |  68 | 68.068 |  6800000000000 | t
 row_69  |  69 | 69.069 |  6900000000000 | f
 row_7   |   7 |  7.007 |   700000000000 | f
 row_70  |  70 |  70.07 |  7000000000000 | t
 row_71  |  71 | 71.071 |  7100000000000 | f
 row_72  |  72 | 72.072 |  7200000000000 | t
 row_73  |  73 | 73.073 |  7300000000000 | f
 row_74  |  74 | 74.074 |  7400000000000 | t
 row_75  |  75 | 75.075 |  7500000000000 | f
 row_76  |  76 | 76.076 |  7600000000000 | t
 row_77  |  77 | 77.077 |  7700000000000 | f
 row_78  |  78 | 78.078 |  7800000000000 | t
 row_79  |  79 | 79.079 |  7900000000000 | f
 row_8   |   8 |  8.008 |   800000000000 | t
 row_80  |  80 |  80.08 |  8000000000000 | t
 row_81  |  81 | 81.081 |  8100000000000 | f
 row_82  |  82 | 82.082 |  8200000000000 | t
 row_83  |  83 | 83.083 |  8300000000000 | f
 row_84  |  84 | 84.084 |  8400000000000 | t
 row_85  |  85 | 85.085 |  8500000000000 | f
 row_86  |  86 | 86.086 |  8600000000000 | t
 row_87  |  87 | 87.087 |  8700000000000 | f
 row_88  |  88 | 88.088 |  8800000000000 | t
 row_89  |  89 | 89.089 |  8900000000000 | f
 row_9   |   9 |  9.009 |   900000000000 | f
 row_90  |  90 |  90.09 |  9000000000000 | t
 row_91  |  91 | 91.091 |  9100000000000 | f
 row_92  |  92 | 92.092 |  9200000000000 | t
 row_93  |  93 | 93.093 |  9300000000000 | f
 row_94  |  94 | 94.094 |  9400000000000 | t
 row_95  |  95 | 95.095 |  9500000000000 | f
 row_96  |  96 | 96.096 |  9600000000000 | t
 row_97  |  97 | 97.097 |  9700000000000 | f
 row_98  |  98 | 98.098 |  9800000000000 | t
 row_99  |  99 | 99.099 |  9900000000000 | f
(100 rows)

SELECT name, num FROM h_base_smoke_test_foreign_table WHERE num > 50 ORDER BY name;
  name   | num 
---------+-----
 row_100 | 100
 row_51  |  51
 row_52  |  52
 row_53  |  53
 row_54  |  54
 row_55  |  55
 row_56  |  56
 row_57  |  57
 row_58  |  58
 row_59  |  59
 row_60  |  60
 row_61  |  61
 row_62  |  62
 row_63  |  63
 row_64  |  64
 row_65  |  65
 row_66  |  66
 row_67  |  67
 row_68  |  68
 row_69  |  69
 row_70  |  70
 row_71  |  71
 row_72  |  72
 row_73  |  73
 row_74  |  74
 row_75  |  75
 row_76  |  76
 row_77  |  77
 row_78  |  78
 row_79  |  79
 row_80  |  80
 row_81  |  81
 row_82  |  82
 row_83  |  83
 row_84  |  84
 row_85  |  85
 row_86  |  86
 row_87  |  87
 row_88  |  88
 row_89  |  89
 row_90  |  90
 row_91  |  91
 row_92  |  92
 row_93  |  93
 row_94  |  94
 row_95  |  95
 row_96  |  96
 row_97  |  97
 row_98  |  98
 row_99  |  99
(50 rows)

-- clean up HBase
{{ CLEAN_UP }}\!{{ HBASE_CMD }} shell {{ SCRIPT drop_small_data.rb }} >/dev/null 2>&1
{{ CLEAN_UP }}\!rm -rf {{ SCRIPT drop_small_data.rb }} {{ SCRIPT gen_small_data.rb }} {{ SCRIPT create_pxflookup.rb }}
